libraries/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -483,13 +483,18 @@ static IotMqttError_t _deserializeIncomingPacket( _mqttConnection_t * pMqttConne
                                                   _mqttPacket_t * pIncomingPacket )
 {
     IotMqttError_t status = IOT_MQTT_STATUS_PENDING;
+    bool packetValidity = false;
+
+    /* Packet validity is not checked when asserts are disabled. */
+    ( void ) packetValidity;
```

Comment:
packetValidity is used on line 496, which is not part of an assert statement, so (void) may not be required.

Reply:
I was following the pattern I observed [here](https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/v4_beta/libraries/standard/mqtt/src/iot_mqtt_operation.c#L717), but after testing with Travis, it doesn't look like there are any build warnings without the voids. I will remove them.